src/main/java/com/salaboy/conferences/site/repository/ProposalRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.salaboy.conferences.site.repository;
+
+import com.salaboy.conferences.site.model.Proposal;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ProposalRepository extends JpaRepository<Proposal, String> {}
```

Comment:
@mcruzdev status filter can be added here in the same way as the findAllByDay was added to the AgendaItemRepo